src/main/webapp/view/discussion/discussion.js

```diff
@@ -12,26 +12,56 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import MarkAnsweredUtil from './mark-answered-util.js';
+
 export const COMMENT_TYPE_REPLY = 'REPLY';
 export const COMMENT_TYPE_QUESTION_UNANSWERED = 'QUESTION_UNANSWERED';
 export const COMMENT_TYPE_QUESTION_ANSWERED = 'QUESTION_ANSWERED';
 export const COMMENT_TYPE_NOTE = 'NOTE';
 
+/**
+ * Provides information on a comment type. For instance: its name,
+ * display styles, and if it can be changed.
+ * @typedef {Object} CommentTypeInfo
+ * @property {string} name The display name of the type.
+ * @property {!Array<string>} badgeStyles List of classes to add to the type
+ *     tag. TODO: Rename to tagStyles.
```

Comment:
TODOs should not be part of the JsDoc. When reading public api documents you never see TODOs. Just add the TODO after the JavaDoc and use // to make it a comment.

Reply:
Changed, thanks!